includes/class-swedbank-pay-payment-gateway-checkout.php

```diff
@@ -127,42 +142,43 @@ public function __construct() {
 		// Load the settings.
 		$this->init_settings();
 
-		// Update access_token if merchant_token is exists
+		// Update access_token if merchant_token is exists.
 		if ( empty( $this->settings['access_token'] ) && ! empty( $this->settings['merchant_token'] ) ) {
 			$this->settings['access_token'] = $this->settings['merchant_token'];
 			$this->update_option( 'access_token', $this->settings['access_token'] );
 		}
 
-		// Define user set variables
-		$this->enabled         = isset( $this->settings['enabled'] ) ? $this->settings['enabled'] : 'yes';
-		$this->title           = isset( $this->settings['title'] ) ? $this->settings['title'] : '';
-		$this->description     = isset( $this->settings['description'] ) ? $this->settings['description'] : '';
-		$this->access_token    = isset( $this->settings['access_token'] ) ? $this->settings['access_token'] : $this->access_token;
-		$this->payee_id        = isset( $this->settings['payee_id'] ) ? $this->settings['payee_id'] : $this->payee_id;
-		$this->testmode        = isset( $this->settings['testmode'] ) ? $this->settings['testmode'] : $this->testmode;
-		$this->ip_check        = defined( 'SWEDBANK_PAY_IP_CHECK' ) ? SWEDBANK_PAY_IP_CHECK : $this->ip_check;
-		$this->culture         = isset( $this->settings['culture'] ) ? $this->settings['culture'] : $this->culture;
-		$this->logo_url        = isset( $this->settings['logo_url'] ) ? $this->settings['logo_url'] : $this->logo_url;
-		$this->instant_capture = isset( $this->settings['instant_capture'] ) ? $this->settings['instant_capture'] : $this->instant_capture;
-		$this->terms_url       = isset( $this->settings['terms_url'] ) ? $this->settings['terms_url'] : get_site_url();
-		$this->autocomplete    = isset( $this->settings['autocomplete'] ) ? $this->settings['autocomplete'] : 'no';
+		// Define user set variables.
+		$this->enabled             = $this->settings['enabled'] ?? 'yes';
+		$this->title               = $this->settings['title'] ?? '';
+		$this->description         = $this->settings['description'] ?? '';
+		$this->access_token        = $this->settings['access_token'] ?? $this->access_token;
+		$this->payee_id            = $this->settings['payee_id'] ?? $this->payee_id;
+		$this->testmode            = $this->settings['testmode'] ?? $this->testmode;
+		$this->ip_check            = defined( 'SWEDBANK_PAY_IP_CHECK' ) ? SWEDBANK_PAY_IP_CHECK : $this->ip_check;
+		$this->culture             = $this->settings['culture'] ?? $this->culture;
+		$this->logo_url            = $this->settings['logo_url'] ?? $this->logo_url;
+		$this->instant_capture     = $this->settings['instant_capture'] ?? $this->instant_capture;
+		$this->terms_url           = $this->settings['terms_url'] ?? get_site_url();
+		$this->autocomplete        = $this->settings['autocomplete'] ?? 'no';
+		$this->exclude_order_lines = wc_string_to_bool( $this->settings['exclude_order_lines'] ?? 'no' );
```

Comment:
Issue: The class property is set as a string type with the default being `'no'`, but this sets it as a bool instead. I would recommend setting the class property type to bool and changing the default to false to match this, since having it as a bool makes it easier to check later.